lib/git_worktree.rb

```diff
@@ -359,9 +377,9 @@ def open_repo
     @repo = Rugged::Repository.new(@path)
   end
 
-  def clone(url)
+  def clone(url, clone_bare)
```

Comment:
variable name wise, the `clone_` prefix seems redundant.  Can this just be `bare` ?

Reply:
🐻